### DISCUSSION TITLE
Skipping results/every nth row

### DISCUSSION THREAD

**Discussion by afcode123** [prisma/prisma#9639] [Q&A] [answered]
How could I perform query in postgres using prisma where every nth row of results is skipped? I have date column in the form of YYYY-DD-MM in database and I am trying to get rows in time intervals for example one row for each week. I could probably use raw query but if anyone has idea how to do this with prisma that would be helpful

**Comment by ryands17** [comment, marked as answer]
@afcode123 👋 

I don't think this is directly possible with Prisma. You would need to fetch the records and omit/skip them in your application code.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
